{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\n7allure_测试报告.py:21: Exception"}, "start": 1728998716547, "stop": 1728998716548, "uuid": "451f7ebe-29d5-4764-aa0a-92772d333c93", "historyId": "0c15b946cfd3cb0a1c1492b3b52d1f11", "testCaseId": "0c15b946cfd3cb0a1c1492b3b52d1f11", "fullName": "7allure_测试报告#test_broken", "labels": [{"name": "suite", "value": "7allure_测试报告"}, {"name": "host", "value": "DESKTOP-HA7F4SE"}, {"name": "thread", "value": "12972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "7allure_测试报告"}]}